spec/controllers/api/irs_attempts_api_controller_spec.rb

```diff
@@ -31,6 +31,21 @@
   let(:existing_event_jtis) { existing_events.map(&:first) }
 
   describe '#create' do
+    context 'with CSRF protection enabled' do
+      around do |ex|
+        ActionController::Base.allow_forgery_protection = true
+        ex.run
+        ActionController::Base.allow_forgery_protection = false
+      end
```

Comment:
```suggestion
        ex.run
      ensure
        ActionController::Base.allow_forgery_protection = false
      end
```
Maybe move that to an `ensure` block just in case something throws an error